lib/yt/models/video.rb

```diff
@@ -247,6 +247,12 @@ def licensed?
         content_detail.licensed_content || false
       end
 
+      # @return [Boolean] whether the video is identified by YouTube as
+      #   age-restricted content.
+      def age_restricted?
+        content_detail.content_rating != nil
```

Comment:
Hello @jacknguyen and thanks for your contribution! 
The source code of yt is a little hard to grasp at first, and you went right for it! :clap: 

Regarding this line, you are stating that having _any_ value in content rating means that the video is "age-restricted". 

However there are [many different values according to the docs](https://developers.google.com/youtube/v3/docs/videos#contentDetails.contentRating).
For instance, the value might indicate that:
> These programs are intended for children (other than preschool children) who are younger than 14 years of age.

In that case, the video is not really "restricted" to an age, just recommended.
Could you elaborate on a case where you needed to know if there was _any_ content-rating in place, independently of the value of the content rating itself?